database/migrations/2023_03_15_161618_add_fields_to_families.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('families', function (Blueprint $table) {
+            $table->timestamp('rejoin_on')->nullable();
+            $table->integer('leave_amount')->default(0);
```

Comment:
Does this need to be added to the Model? ($fillable)

Reply:
Done